opm/input/eclipse/Schedule/Network/NetworkKeywordHandlers.cpp

```diff
@@ -78,12 +78,13 @@ void handleGRUPNET(HandlerContext& handlerContext)
         throw OpmInputError(msg, handlerContext.keyword.location());
     }
     network.set_standard_network(true);
-    const char* info_msg =
-        " \n"
-        " PLEASE NOTE: \n"
-        "   Flow writes restart data for standard network in extended network format. \n"
-        "   Restarting other simulators from Flow output requires conversion to extended network. \n"
-        ;
+    const std::string info_msg = fmt::format(
+        "\n"
+        " PLEASE NOTE:\n"
+        "   Flow writes restart data for standard network in extended network format.\n"
+        "   Restarting other simulators from Flow output requires conversion to extended network.\n"
+    );
```

Comment:
If you're really going to do this, then please do at least make the `info_msg` `const`.  On a second note, you don't need `fmt::format()` for this.  You can just use a raw string literal instead:
```C++
const auto info_msg = std::string { R"(
 PLEASE NOTE:
    Flow writes ...
    Restarting other ...
)" };

Reply:
I tried this way, but if I need to keep the original formatting of output, it will be something like the following, 

```c++
    const std::string info_msg = R"(

 PLEASE NOTE:
   Flow writes restart data for standard network in extended network format.
   Restarting other simulators from Flow output requires conversion to extended network.
)";
```

It is kind of interrupting the indentation.

Reply:
> It is kind of interrupting the indentation.

Sure.  It's still the better option.

Reply:
can you have a look and let me know what you think about the current version?